openhtf/io/frontend/templates/station.html

```diff
@@ -30,7 +30,7 @@
             document.getElementById(id).innerHTML = new_doc; });
   }
   function poll(interval_ms) {
```

Comment:
The current setup around poll looks like it's essentially a setInterval, so the equivalent is this:

```js
function poll() {
  ...
}
function setup() {
  setInterval(poll, {{interval}});
}
```

However, this can lead to DoS'ing a slow server (when the server isn't broken but takes longer than interval_ms to respond). Instead, scheduling a new poll only when the last one returned would be better (optionally subtracting the time the poll took to make the interval closer to the poll_interval_ms flag value):

```js
function poll() {
  refresh('test');
  ajax(..., function(id) {
    ...
    setTimeout(poll, {{interval}});
  });
}
function setup() {
  poll();
}
```